pkg/framework/context/context.go

```diff
@@ -225,10 +231,18 @@ func WithPackager(p envelope.Packager) ProviderOption {
 	}
 }
 
-// WithCrypter injects a crypter into the context
-func WithCrypter(p crypto.Crypter) ProviderOption {
+// WithDefaultPacker injects a packer into the context
+func WithDefaultPacker(p packer.Packer) ProviderOption {
+	return func(opts *Provider) error {
+		opts.defaultPacker = p
+		return nil
+	}
+}
+
+// WithPackers injects a variable number of packer services into the Aries framework
+func WithPackers(packers ...packer.Packer) ProviderOption {
```

Comment:
same thing with options, the user now has options: WithPackager, WithDefaultPacker and WithPackers.. it will confuse to the consumer of the framework as to what is required... another reason why Packager interface becomes redundant with this PR.

removing it can be done in a future PR.. add TODOs in the right locations if doing it separately.

Reply:
I've removed framework `WithPackager()`